src/api/status/package.json

```diff
@@ -9,9 +9,7 @@
     "start": "node src/server.js",
     "watch:server": "env-cmd -f env.local nodemon src/server.js",
     "compile:js": "vite build",
-    "watch:js": "vite build --watch",
-    "compile:scss": "sass ./scss/material-dashboard.scss:./public/assets/css/material-dashboard.css",
-    "watch:scss": "sass --watch ./scss/material-dashboard.scss:./public/assets/css/material-dashboard.css"
+    "watch:js": "vite build --watch"
```

Comment:
As I understand this, we're using Vite to transform CommonJS modules into ESM and importing them. Why do need to watch for changes when `pnpm dev` when these dependencies are static?
```json
    "dev": "pnpm build && pnpm watch:server",
```

Reply:
> As I understand this, we're using Vite to transform CommonJS modules into ESM and importing them. Why do need to watch for changes when pnpm dev when these dependencies are static?

Vite also builds our JS files (e.g. terminal.js). We don't want to manually run `vite build` on every js changes.